src/comp.py

```diff
@@ -4,7 +4,13 @@
 x = input("Enter comma-separated numbers: ").split(',')
 
 # What do you need between the square brackets to make it work?
-y = []
+def even(): 
```

Comment:
Same ends but the means are off.

We need to preform this operation inline to the brackets such as...
https://stackoverflow.com/questions/10973766/understanding-the-map-function#10973817
You'll see the pattern [func(i) for i in iterable] and [(a, b) for a in iterable_a for b in iterable_b]